src/modules/datepicker/helpers/date-fns.ts

```diff
@@ -13,20 +13,28 @@ interface IDateFnsCustomLocale {
         weekdays:DateFnsHelper<IDateFnsHelperOptions, string[]>;
         month:DateFnsHelper<number, string>;
         months:DateFnsHelper<IDateFnsHelperOptions, string[]>;
+        timeOfDay:DateFnsHelper<number, string>;
+        timesOfDay:DateFnsHelper<IDateFnsHelperOptions, string[]>;
     };
     match:{
         weekdays:DateFnsHelper<string, RegExpMatchArray | null>;
         weekday?:DateFnsHelper<RegExpMatchArray, number>;
         months:DateFnsHelper<string, RegExpMatchArray | null>;
         month?:DateFnsHelper<RegExpMatchArray, number>;
+        timesOfDay:DateFnsHelper<string, RegExpMatchArray | null>;
+        timeOfDay?:DateFnsHelper<RegExpMatchArray, number>;
     };
     options?:{
         weekStartsOn?:number;
     };
 }
 
-function buildLocalizeFn(values:IDateFnsLocaleValues, defaultType:string):DateFnsHelper<number, string> {
-    return (dirtyIndex, { type } = { type: defaultType }) => values[type][dirtyIndex];
+function buildLocalizeFn(values:IDateFnsLocaleValues, defaultType:string,
+                         indexCallback?:(oldIndex:number) => number):DateFnsHelper<number, string> {
```

Comment:
Could you put each arg on its own line please? Leave a space before the 1st statement in the function also when breaking like this